### PR TITLE
[WIP] Change realization oauth2 from js library to dirac service(associated with OAuthDIRAC)

### PR DESCRIPTION
**Pull request by TaykYoku**
BEGINRELEASE

*Lib/Conf.py
CHANGE: Added configuration helpers for authorisation handler

*Lib/WebHandler.py
FIX: Fixed some bugs in oauth2 authorisation logic

*WebApp/handler/AuthenticationHandler.py
CHANGE: Changed logic to work with oauth2 service

*WebApp/static/core/js/views/desktop/TaskBar.js
CHANGE: Changed logic to work with oauth2 service

*WebApp/static/oidc
FIX: Removed

*WebApp/template/root.tpl
FIX: Removed OIDC js library

ENDRELEASE

### CODE REVIEW

**Comment by TaykYoku** [comment]
[v0r1-pre2]OAuthDIRAC extension depends on this PR

File: WebApp/static/core/js/views/desktop/TaskBar.js [hunk 1]
```diff
@@ -166,24 +168,28 @@ Ext.define('Ext.dirac.views.desktop.TaskBar', {
     };
 
     var auth = function(AuthType) {
-      result = getAuthCFG(AuthType,'method')
+      console.log('"' + AuthType + '" was choosed to authorization')
```
Comment (fstagni) [inline WebApp/static/core/js/views/desktop/TaskBar.js]: "chosen for authorization"

File: WebApp/static/core/js/views/desktop/TaskBar.js [hunk 2]
```diff
@@ -166,24 +168,28 @@ Ext.define('Ext.dirac.views.desktop.TaskBar', {
...
+      result = getAuthCFG(AuthType, 'Type')
       if (!result.OK) { return GLOBAL.APP.CF.alert("Authentication was ended with error: %s", result.Message,'error') }
       if (result.Value) {
+        console.log('"' + result.Value + '" is not null')
         Metod = result.Value
-        if (Metod == 'oAuth2') {
-          result = getAuthCFG(AuthType,'authorization_url')
+        if (Metod == 'OAuth2') {
```
Comment (fstagni) [inline WebApp/static/core/js/views/desktop/TaskBar.js]: Metod == method?

File: WebApp/static/core/js/views/desktop/TaskBar.js [hunk 3]
```diff
@@ -166,24 +168,28 @@ Ext.define('Ext.dirac.views.desktop.TaskBar', {
...
-          console.log('========authorization_url=========')
-          console.log(authorization_url)
+          
           //Open popup
+          console.log('Open authorization URL: "' + authorization_url + '"')
           var oAuthReqWin = open(authorization_url, "popupWindow", "hidden=yes,height=570,width=520,scrollbars=yes,status=yes")
           oAuthReqWin.focus();
           
-          // Send request to redirect_uri about success auth
+          // Send request to redirect URL about success authorzation
```
Comment (fstagni) [inline WebApp/static/core/js/views/desktop/TaskBar.js]: authorization

**Comment by fstagni** [comment]
The branch has conflicts now, please rebase or merge.

**Pull request closed without merging**